mesheryctl/internal/cli/root/system/context/delete.go

```diff
@@ -0,0 +1,32 @@
+package context
+
+import (
+	"errors"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// deleteContextCmd represents the update command
+var deleteContextCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "delete context",
+	Long:  `delete specified context from config`,
+	Args:  cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		log.Printf("Delete Context %s from config", args[0])
```

Comment:
Are we checking for the presence of an argument and providing usage suggestion if no arg is included?

Reply:
Cobra handles that

Reply:
Ok, sounds good.